hw/kdrive/src/kdrive.h

```diff
@@ -217,7 +217,7 @@ void KdRemovePointerDriver(KdPointerDriver * driver);
 KdPointerInfo *KdNewPointer(void);
 void KdFreePointer(KdPointerInfo *);
 int KdAddPointer(KdPointerInfo * ki);
-int KdAddConfigPointer(char *pointer);
+int KdAddConfigPointer(const char *pointer);
```

Comment:
will merge this into maint-25.0 via https://github.com/X11Libre/xserver/pull/357